.github/workflows/integration-tests.yml

```diff
@@ -10,6 +10,11 @@ jobs:
       - uses: DeterminateSystems/nix-installer-action@main
       - uses: DeterminateSystems/magic-nix-cache-action@main
       - uses: Swatinem/rust-cache@v2
+      - name: Install neovim
+        uses: rhysd/action-setup-vim@v1 # Used by the 'run' integration test
+        with:
+          neovim: true
+          version: v0.10.2
```

Comment:
I got a function laying around somewhere to automatically get the latest tag
let me find this rq

Reply:
up to you if you want to add it
https://github.com/max397574/care.nvim/blob/61f564c71ac3642f4297428e3bdb03af6b1ebd4a/.github/workflows/typecheck.yml#L14

Reply:
Actually, for this workflow if you omit `version`, it'll install the latest stable version.